docs/make.jl

```diff
@@ -11,6 +11,9 @@ using QEDbase
 
 #DocMeta.setdocmeta!(QEDbase, :DocTestSetup, :(using QEDbase); recursive=true)
 
+# TODO: remove before release
+Pkg.add(; url="https://github.com/QEDjl-project/QEDcore.jl", rev="dev")
+
```

Comment:
Does this need to be removed now?

Reply:
Good question! I think we leave it there for now, because `QEDcore` must be released for this to work, and `QEDcore` can be released only if this is released in the first place 🤷‍♂️

Reply:
Yes, I think you're right. Then we should come back to this once QEDcore is released as well.